File: 第一期/java作业/Java文档/类的无参方法补充案例  源代码参考/Book.java
/**
 * 以面向对象的思想，编写自定义类描述图书信息 
 * 设定属性包括：书名，作者，出版社名，价格 
 * 方法包括：信息介绍 
 */
public class Book {
	public String bookName;// 书名
	public String author;// 作者
	public String publisher;// 出版社
	public double price;// 价格

	// 信息介绍方法描述图书所有信息
	public void info() {
		String content = "书名：" + this.bookName + "\n作者：" + this.author + "\n出版社:"
				+ this.publisher + "\n价格：" + this.price+"元";
		System.out.println(content);
	}
}

File: 第三期/使用ASP.NET技术开发网上书店/项目/BookNet/BookNetModels/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookNet.Models
{
    [Serializable]
    public class Book
    {
        /// <summary>
        /// 书籍编号
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// 书籍名称
        /// </summary>
        public string BookName { get; set; }

        /// <summary>
        /// 作者
        /// </summary>
        public string Author { get; set; }

        /// <summary>
        /// 出售单价
        /// </summary>
        public decimal SellMoney { get; set; }

        /// <summary>
        /// 数量
        /// </summary>
        public int Number { get; set; }

        /// <summary>
        /// 公司购书时间
        /// </summary>
        public DateTime BuyDate { get; set; }

        /// <summary>
        /// 书籍发布时间
        /// </summary>
        public DateTime PublisherDate { get; set; }

        /// <summary>
        /// 评论
        /// </summary>
        public Comment Comment { get; set; }

        /// <summary>
        /// 出版社
        /// </summary>
        public Publisher Publisher { get; set; }

        /// <summary>
        /// 书籍类型
        /// </summary>
        public Mold Mold { get; set; }

        /// <summary>
        /// 内容简介
        /// </summary>
        public string Content { get; set; }

        /// <summary>
        /// 书籍状态<true:完本，false:未完本>
        /// </summary>
        public bool BookState { get; set; }

        /// <summary>
        /// 字数
        /// </summary>
        public int WordCount { get; set; }

        /// <summary>
        /// 书籍封面图片
        /// </summary>
        public string Picture { get; set; }

        /// <summary>
        /// 点击率
        /// </summary>
        public int Clicks { get; set; }
    }
}
